source/index.html

```diff
@@ -1,77 +1,125 @@
+                    <tr><img class="social-icon" src="./assets/social/yelp-2-24.png"></tr>
                 </table>
             </div>
-            <div class="content-box">
-                <a href="http://goo.gl/maps/7xS1a">
-                    <img src="./images/panara-imports-static-map.png" alt="" width="300" height="300">
-                </a>
+        </nav>
+        <img class="logo" src="./assets/panara-logo.svg" />
+        <h1 class="title">Panara Imports</h1>
+        <h3 class="sub-title">UNIQUE FASHIONS &amp; FABULOUS FINDS SINCE 1992</h3>
+        <p class="email-phone">
+            <a href="mailto:panaramqt@gmail.com">panaramqt@gmail.com</a>
+            <span><a href="tel:906-226-4604">(906) 226-4604</a></span>
+        </p>
+        <p class="email-phone"> Mon - Fri 10-7 <span>Sat 12-7</span><span>Sun 12-4</span>
+        </p>
+    </section>
+    <section class="location">
+        <h1>Location</h1>
+        <h3>Great atmosphere very relaxed environment and the nicest changing rooms ever!</h3>
+        <p><a href="#">Directions</a><span>125 W Washington St Marquette MI 49855</span></p>
+        <p><a href="#">Event Calendar</a></p>
+        <div class="winter-surf">
+            <img src="./assets/winter-surf.png">
+            <p><strong>We have you covered!</strong></p>
+            <p>No matter the weather.</p>
+        </div>
+        <div class="alpaca_sweater">
+            <img src="./assets/alpaca_sweater.png">
+            <p><strong>Open 7 days a week</strong></p>
+            <p><strong>362 Days a year!</strong></p>
+        </div>
+        <div class="relaxed">
+            <img src="./assets/relax-enviroment.png">
+            <p><strong>Relaxing enviroment!</strong></p>
+            <p>Come in and shop in comfort.</p>
+        </div>
+    </section>
+    <img class="img-divider" src="./assets/landscape-1.png">
+    <section class="shopping">
+        <h1>Shopping</h1>
+        <h3>Can't get into the store? You can find most of your favorites like incense and body oils in our online store.</h3>
+        <p><a href="#">Online Store</a></p>
+        <p><a href="#">Fashion Revolution</a></p>
+            <img class="overlay" src="./assets/who-made-my-cloths.png">
+            <div class="panara-shirt">
+                <p><strong>Panara Merchandise</strong></p>
+                <p>great in any weather</p>
+                <img src="./assets/panara-shirt.png">
             </div>
-        </main>
-    </body>
+            <div class="black-jacket">
+                <img src="./assets/black-jacket.png">
+                <p><strong>Cloths and accesories for any occasion</strong></p>
+                <p>check daily for updates and new products</p>
+            </div>
+            <div class="oils">
+                <img src="./assets/oils.png">
+                <p><strong>Esenial Oils and Incents</strong></p>
+                <p>available year round instore or online</p>
```

Comment:
Splitting a sentence up over multiple `p` tags seems weird to me … oh, I see. It's probably more like the first `p` is actually a heading of some kind? Maybe `h5` or `h6` would be more appropriate?